src/Functions/randomStringUTF8.cpp

```diff
@@ -153,6 +153,27 @@ class FunctionRandomStringUTF8 : public IFunction
 
 REGISTER_FUNCTION(RandomStringUTF8)
 {
-    factory.registerFunction<FunctionRandomStringUTF8>();
+    FunctionDocumentation::Description description = R"(
+Generates a random string of a specified length.
+The resulting string contains valid UTF-8 code points.
+The value of code points may be outside of the range of assigned Unicode.
+    )";
+    FunctionDocumentation::Syntax syntax = "randomStringUTF8(length)";
+    FunctionDocumentation::Arguments arguments = {
+        {"length", "Length of the string in code points.", {"U)Int*", "Float*"}}
```

Comment:
The cold, hard truth is that the function accepts `Decimal*` as well. This is really silly and should not have happened in the first place. Anyways, it is too late now. Can we ~lie~ not tell the full truth and just say that `(U)Int*` is supported?

Reply:
“Above all, don't lie to yourself. The man who lies to himself and listens to his own lie comes to a point that he cannot distinguish the truth within him, or around him, and so loses all respect for himself and for others. And having no respect he ceases to love.” - Fyodor Dostoevsky

He didn't mention _not telling the full truth_ though, so I think we're good!